src/main/resources/native/android/c/grandroid.h

```diff
@@ -47,4 +74,6 @@ void __attribute__((weak)) androidJfx_setNativeWindow(ANativeWindow *nativeWindo
 void __attribute__((weak)) androidJfx_setDensity(float nativeDensity) {}
 void __attribute__((weak)) androidJfx_gotTouchEvent(int count, int *actions, int *ids, int *xs, int *ys, int primary) {}
 void __attribute__((weak)) androidJfx_gotKeyEvent(int action, int key, jchar *chars, int count, int mods) {}
-int  __attribute__((weak)) to_jfx_touch_action(int state) { return 0; }
\ No newline at end of file
+int  __attribute__((weak)) to_jfx_touch_action(int state) { return 0; }
+
+void __attribute__((weak)) attach_setLifecycleEvent(const jchar *chars, int count) {}
```

Comment:
Don't pass java types in between native layers.
This better done as 
`void __attribute__((weak)) attach_setLifecycleEvent(const char *chars)`